js/node/src/directml_load_helper.cc

```diff
@@ -5,37 +5,40 @@
 #include "common.h"
 #include "windows.h"
 
-extern "C" __declspec(dllexport) void ExportFunction()
+void LoadDirectMLDll(Napi::Env env)
```

Comment:
How heavyweight is passing `Env` by value copy? I see other places pass an `Env` by constant reference. e.g.

`Session(const Env& env, const ORTCHAR_T* model_path, const SessionOptions& options);`

Reply:
Should not be heavy as Node.js passes it by copy to the wrapper library. But will change it anyway